controllers/apps/v1beta3/emqxplugin_controller.go

```diff
@@ -117,31 +117,21 @@ func (r *EmqxPluginReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			return ctrl.Result{}, err
 		}
 	}
-
-	for _, emqx := range emqxList {
-		ready := false
-		for _, c := range emqx.GetStatus().Conditions {
-			if c.Type == appsv1beta3.ConditionRunning && c.Status == corev1.ConditionTrue {
-				ready = true
-			}
-		}
-		if !ready {
-			break
-		}
-
-		equalPluginConfig, err := r.checkPluginConfig(instance, emqx)
-		if err != nil {
+	ready, emqx := findReadyEmqx(emqxList)
```

Comment:
Why didn't use `handler. findReadyEmqx` ?

Reply:
the parameters are different